pypistats/cli.py

```diff
@@ -103,15 +103,36 @@ def _valid_yyyy_mm(date_string):
     [
         argument("package"),
         argument("-p", "--period", choices=("day", "week", "month")),
-        argument("-j", "--json", action="store_true", help="Output JSON"),
+        argument(
+            "-j",
+            "--json",
+            action="store_true",
+            help='Output JSON. Shortcut for "-f json".',
+        ),
+        argument(
+            "-f",
+            "--format",
+            default="markdown",
+            help="The format of output. Supported: "
+            f'{", ".join(pypistats.FORMATS)}. '
+            'Default is "markdown".',
+        ),
     ]
 )
 def recent(args):
-    print(
-        pypistats.recent(
-            args.package, period=args.period, output="json" if args.json else "table"
-        )
-    )
+    # "table" means "markdown". legacy.
+
+    if args.json or args.format == "json":
+        output = "json"
+    elif args.format == "markdown":
+        output = "table"
+    else:
+        import warnings
```

Comment:
Let's move the import to the top and use `isort` to put them in order.

Reply:
Alright.